firmware/Components.cpp

```diff
@@ -350,9 +350,68 @@ char * AtlasPH::name(){
 }
 
 bool AtlasPH::set(const char* param, const char* value){
-  if (strncmp(param, "temp", 4) == 0){
-    this->setTemp(atof(value));
-    return true;
+  if (strncmp(param, "t", 1) == 0){
+    if ( value != NULL )
+    {
+      this->setTemp(atof(value));
+      return true;
+    }
+    //Send temperature compensation value
+    else 
+    {
+      char output_str[DEFAULT_BUFFER_SIZE + 3];
+      snprintf(output_str, DEFAULT_BUFFER_SIZE,
+               "{"
+               "\"s%u\":{"
+               "\"type\":\"%s\","
+               "\"data\":\t,%f\""
+               "}"
+               "}",
+               channel_,
+               this->name(),
+               temperature
+              );
+      send(output_str);
+        
+    }
+  } else if (strncmp(param, "c", 1) == 0){
+      //Get calibration status
+      if (value == NULL)
+      {
+        char output_str[DEFAULT_BUFFER_SIZE + 3];
+      snprintf(output_str, DEFAULT_BUFFER_SIZE,
+               "{"
+               "\"s%u\":{"
+               "\"type\":\"%s\","
+               "\"data\":\"c,%f\""
+               "}"
+               "}",
+               channel_,
+               this->name(),
+               calibrationStatus
+              );
+      send(output_str);
+         
```

Comment:
Remove whitespace, fix indent (this might be a spaces/tabs mix-- if so, convert everything to spaces).